uldk_api.py

```diff
@@ -41,3 +41,11 @@ def getVoivodeshipById(id, objectType):
     res = Request(params, objectType)
     return res.data
 
+
+def get_building_by_id(id, object_type):
```

Comment:
Olu, zmień tylko na getBuildingById, żeby zachować jednolitą konwencję, tak to bardzo ładnie napisane :)